socorro/processor/rules/general.py

```diff
@@ -96,13 +96,6 @@ class CPUInfoRule(Rule):
 
     """
 
-    # Map of Android_CPU_ABI values to cpu_arch values
-    ANDROID_CPU_ABI_MAP = {
-        "armeabi-v7a": "arm",
-        "arm64-v8a": "arm64",
-        "x86_64": "amd64",
-    }
-
```

Comment:
Oops--this needs a predicate so it only kicks in if `json_dump.system_info.cpu_info` exists.

Reply:
Actually, scratch that--the way I have it now, it does exactly what the old code did.